### DISCUSSION TITLE
heat protection not kicking in when calibrating

### DISCUSSION THREAD

**Discussion by ghost** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1277] [Q&A] [answered]
Quick question to everyone: why isn't heat protection kicking in when calibrating?
I'm watching my battery slowly creeping from about 32C all the way to almost 40 while charging under calibration mode. 
My limit is set at 35C, yet heat protection is not working, not even after about 30minutes where my battery slowly reached 40C. 
Why is that? is it supposed to happen? Is it a bug? 
Currently using a Macbook Pro 16'' with M2

Thanks

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @antevim1,

We decided to disable heat protection during calibration mode to not prolong the calibration even further. Also, since the calibration mode is only used roughly every two weeks, the negative side effects are low. Furthermore, we have noticed a slightly better outcome if the calibration is not interrupted.

Best,
Matthias

**Reply by ghost** [reply]
That explains it, thank you! 
Also, thank you for your hard work!
